sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/automl_vertical.py

```diff
@@ -11,12 +11,48 @@
 
 
 class AutoMLVertical(AutoMLJob):
-    """Abstract class for AutoML verticals."""
+    """Abstract class for AutoML verticals.
+
+    :param task_type: The type of task to run. Possible values include: "classification", "regression", "forecasting".
```

Comment:
same as above for the replication

Reply:
Both the class and constructor must be documented.
__init__ docstring is not visible on the learning docs portal, but the docstring from init method is extract on running `help(YourClassName. __init__)`. A good strategy to follow is to make your changes in the class docstring and then copy the changes to the __init__ method docstring.